vignettes/epidist.Rmd

```diff
@@ -1,201 +1,211 @@
 ---
-title: "Getting Started with Epidist"
-description: "A quick start guide to using the Epidist package"
-author: Epidist Team
+title: "Getting started with epidist"
+description: "A quick start guide to using the epidist R package"
 output: 
-  bookdown::html_vignette2:
+  bookdown::html_document2:
     fig_caption: yes
     code_folding: show
+    number_sections: true
 pkgdown:
   as_is: true
 csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
 link-citations: true
 vignette: >
-  %\VignetteIndexEntry{Getting Started with Epidist}
+  %\VignetteIndexEntry{Getting started with epidist}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
 ```{r setup, include=FALSE}
 # exclude compile warnings from cmdstanr
 knitr::opts_chunk$set(
-  fig.path = "figures/getting-started-nowcasting-",
-  cache = TRUE, dpi = 330,
-  collapse = TRUE, comment = "#>", out.width = "100%",
-  message = FALSE, warning = FALSE, error = FALSE,
-  eval = FALSE
+  fig.path = "figures/epidist-",
+  cache = TRUE,
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE,
+  warning = FALSE,
+  error = FALSE
 )
 ```
 
-# Quick start
-
-In this quick start, we demonstrate using `epidist` to ...
-
-# Package
-
-In this quick start, we also use `data.table`, `purrr`, and `ggplot2` packages. These are both installed as dependencies when `epidist` is installed. Note that all output from `epidist` is readily useable with  other tools, including `tidyverse` packages (see [here](https://mgimond.github.io/rug_2019_12/Index.html) for a comparison).
-
 ```{r load-requirements}
 library(epidist)
 library(data.table)
 library(purrr)
 library(ggplot2)
 ```
 
-### Simulate the data
+Many quantities in epidemiology can be thought of as the time between two events, or "delays".
+Important examples include:
+
+* the incubation period (time between infection and symptom onset),
+* serial interval (time between symptom onset of infectee and symptom onset of infected), and 
+* generation interval (time between infection of infectee and infection of infected).
+
+Unfortunately, estimating delays accurately from observational data is usually difficult.
+In our experience, the two main issues are:
 
-Simulate data from an outbreak.
+1. interval censoring, and
+2. right truncation.
 
-```{r simulate-outbreak}
+Don't worry if you've not come across these terms before!
+In Section \@ref(data), we will explain what they mean by simulating example data analogous to what we might observe during an ongoing infectious disease outbreak.
+Next, in Section \@ref(fit), we show how `epidist` can be used to estimate delays using a statistical model which properly accounts for these two issues.
+Finally, in Section \@ref(compare), we demonstrate that the fitted delay distribution accurately recovers the underlying truth.
+
+# Example data {#data}
+
+Data should be in a certain format for use within the `epidist` package.
```

Comment:
Here, I'd insert a few lines describing the required data schema, including the required fields, a definition for each field, and notes on data type requirements, e.g.:

```suggestion
Data should be formatted as a [`data.table`]() with the following columns for use within the `epidist` package:

* `case` (data type): a unique ID for each observed case
* `ptime` (data type): ...
* etc. 
```

Reply:
I think this is a good suggestion. My only uncertainty is that I don't exactly know which columns are "hard requirements" for using the package and which are "there sometimes".

I suppose doing this will clarify this point for me (which is needed to do anyway).

Reply:
I still think this is a good suggestion but would like to leave to a later issue as I am unsure of the input requirements we want, and also unsure of the columns we want.